.travis.yml

```diff
@@ -24,49 +27,28 @@ before_install:
       "max-concurrent-uploads": 50
     }' | sudo tee /etc/docker/daemon.json
   - sudo service docker restart
-  # prevent garage from exiting on config
-  - cp garage/config_personal_template.py garage/config_personal.py
+  # Reinstall docker compose with the specified version
```

Comment:
whoah this is strange. why can't we just pull the right version of docker from apt?

Reply:
okay i see this in the Travis documentation.

can you include this link for explanation.

https://docs.travis-ci.com/user/docker/#using-docker-compose

Reply:
Done